### PR TITLE
[RFR] Characteristics with dataset cover - public UI

### PR DESCRIPTION
**Pull request by djhi**
- [x] Introduce `publishedCharacteristics` collection
- [x] Updated `publish` feature to create the publishedCharacteristics entries
- [x] Added public UI components to display them on List page
- [x] e2e tests

screen

### CODE REVIEW

File: src/app/js/admin/fields/sagas/updateField.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ export function* handleUpdateField({ meta: { form } }) {
         yield put(updateFieldError(error));
     } else {
         yield put(updateFieldSuccess(response));
+        yield put(updateField(fieldData));
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/sagas/updateField.js]: updateField name seems wrong, we just uploaded the field successfully, and then we call updateField action?

File: src/app/js/admin/fields/sagas/updateField.js [hunk 2]
```diff
@@ -30,8 +30,8 @@ export function* handleUpdateField({ meta: { form } }) {
         yield put(updateFieldError(error));
```
Comment (ThieryMichel) [inline src/app/js/admin/fields/sagas/updateField.js:30]: add a return to avoid the else

File: src/app/js/dataset/Dataset.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ export class DatasetComponent extends Component {
         return (
             <Card className="dataset">
                 <CardText>
-                    <Table selectable={false} fixedHeader={false} style={{ width: 'auto' }}>
+                    <Table selectable={false} fixedHeader={false} style={{ width: 'auto', minWidth: '100%' }}>
```
Comment (ThieryMichel) [inline src/app/js/dataset/Dataset.js]: No literal declared in render to avoid useless rerendering.

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/app/js/admin/fields/sagas/updateField.js (hunk 2 above)
Comment (djhi) [reply src/app/js/admin/fields/sagas/updateField.js:30]: Done

**Review by ThieryMichel — APPROVED** [review]

**Pull request merged**